textdescriptives/tests/test_extractor.py

```diff
@@ -0,0 +1,45 @@
+import textdescriptives as td
+import spacy
+import pytest
+
+
+@pytest.fixture(scope="function")
+def nlp():
+    nlp = spacy.load("en_core_web_sm")
+    nlp.add_pipe("textdescriptives")
+    return nlp
```

Comment:
You could import this from one script so that it does not need to load it multiple times.

Reply:
the scripts need slightly different functions (for instance, not all require a parser). But yeah, might make things a bit faster